blingfire/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    skeptic::generate_doc_tests(&["../README.md"]);
```

Comment:
For me this fails to link with blingfire

Reply:
that's because the static lib didn't work for you. it should be fine now